README.md

```diff
@@ -45,6 +45,7 @@
 ### Linting
 
 - [eslint-plugin-jest](https://www.github.com/jest-community/eslint-plugin-jest) ESLint plugin for Jest.
+- [eslint-plugin-jest-formatting](https://github.com/dangreenisrael/eslint-plugin-jest-formatting)
```

Comment:
Could you add a quick description as well?

Reply:
sure

Reply:
Done